lib/src/chart/base/base_chart/base_chart_data.dart

```diff
@@ -184,3 +184,15 @@ typedef MouseCursorResolver<R extends BaseTouchResponse> = MouseCursor Function(
 abstract class BaseTouchResponse {
   BaseTouchResponse();
 }
+
+/// Controls showing tooltip horizontal alignment relative to the rod/spot.
+enum FLTooltipAlign {
```

Comment:
I think it's better to use `FlHolrizontalAlign` which is a generalized name to use anywhere (not only tooltip). What do you think?

Reply:
Good idea, will fix it.